edenai_apis/apis/affinda/client.py

```
@@ -518,3 +520,113 @@ def delete_document(self, identifier: str) -> None:
             url=f"{self.BASE_URL}/documents/{identifier}",
             headers=self.headers,
         )
+
+    ######### ASYNC #########
+    async def __arequests(
```

Pylint (4.0.2)

[refactor] Too many arguments (8/7)

(R0913)